index.html

```diff
@@ -1196,6 +1196,12 @@ <h3>Identifiers</h3>
           </li>
         </ul>
 
+        <p class="note">
+If a credential contains a credentialSubject property and it, in turns 
```

Comment:
Per [section 4.4](https://w3c.github.io/vc-data-model/#credential-subject) a credential MUST contain a credentialSubject property. I think we can strike this first line since it's not a branching point.